builder-support/specs/pdns-recursor.spec

```diff
@@ -104,9 +91,6 @@ EOF
 # The EL7 and 8 systemd actually supports %t, but its version number is older than that, so we do use seperate runtime dirs, but don't rely on RUNTIME_DIRECTORY
```

Comment:
should this comment go?

Reply:
yes! I think that's what ph1 spotted too

Reply:
ah yes!